File: Latihan 2/bahasapython/Hardware.py
from Product import Product

class Hardware(Product):
#kelas yang digunakan untuk mengimplementasikan kelas hardware

    def __init__(self):
    #konstruktor
        self.brand = ""
        self.model = ""
        
    def setBrand(self, brand):
    #mengeset nilai atribut brand
        self.brand = brand
        
    def getBrand(self):
    #mengembalikan nilai atribut brand
        return self.brand
    
    def setModel(self, model):
    #mengeset nilai atribut model
        self.model = model
        
    def getModel(self):
    #mengembalikan nilai atribut model
        return self.model
        

File: Latihan 1/bahasajava/Hardware.java
//create derived class 1 - intermediary
class Hardware extends Product{
	
	//private instance variables
	private String brand;
	private String model;
	
	//constructors
	Hardware(){
	}

	//public getter and setter for the private variable brand	
	public void setBrand(String brand){
		this.brand = brand;
	}
	//return variable brand
	public String getBrand(){
		return this.brand;
	}
	
	//public getter and setter for the private variable model		
	public void setModel(String model){
		this.model = model;
	}
	//return variable model
	public String getModel(){
		return this.model;
	}
	
}
	
